lib/galaxy/managers/workflows.py

```diff
@@ -388,12 +388,12 @@ def _workflow_to_dict_run( self, trans, stored ):
                     'name'   : step.module.name,
                     'inputs' : [ input.to_dict( trans ) for input in inputs.itervalues() ]
                 }
-            step_model[ 'step_id' ] = step.id
+            step_model[ 'step_id' ] = trans.app.security.encode_id( step.id )
```

Comment:
Yeah whatever uniquely identifies the step would work. I used the id because its native to the WorkflowStepConnection model.